doc/parameters/param_list.md

```diff
@@ -131,8 +133,8 @@ Namespace: /parent_ns/local_ns
 | Parameter | Default | Description |
 | ----------| --------| ------------|
 | controller_frequency | 20.0 | Frequency to run controller |
-| controller_plugin_ids | ["FollowPath"] | List of mapped names for controller plugins for processing requests and parameters |
-| controller_plugin_types | ["dwb_core::DWBLocalPlanner"] | List of registered plugins to load |
+| controller_plugins | ["FollowPath"] | List of mapped names for controller plugins for processing requests and parameters |
+| FollowPath.plugin | "dwb_core::DWBLocalPlanner" | Default controller plugin |
```

Comment:
Replace "FollowPath" with "<controller plugin>" or something, since there's no requirement to use that namespace. See the other plugins for examples of how we do that

Reply:
Ditto for all below

Reply:
Mentioning something like `<controller plugin>.plugin` doesn't sit right with me since technically we are only providing default values for the `FollowPath` namespace. I've added a small note for all the servers mentioning all the default namespaces and plugins being loaded when the `plugins` parameter is not overridden. That should be enough I think?

Reply:
I think we should add to the yaml and website the fact that the `<plugin namespace>.plugin` exists, because right now its not explicitly mentioned in any of the documentation. There's the param for namespace but not a specific mention that within that namespace, a `plugin` param must exist containing the plugin registered name. So I think what you have right now is good, but we should add to somewhere a subsection like you did for the default values explaining that.